client/extensions/zoninator/state/data-layer/feeds/util.js

```diff
@@ -1,3 +1,9 @@
-export const fromApi = data => data.map( post => post.ID );
+export const fromApi = posts => posts.map( post => ( {
+	ID: post.ID,
+	title: post.post_title,
+	URL: post.guid,
```

Comment:
`ID` and `URL` should be lower case as per https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/javascript.md#naming-conventions

Reply:
It's actually purposefully kept as ID and URL so it matches the results returned by `QueryPosts` component because the two are mixed once you start adding posts to a zone.

Reply:
... But I guess I could convert those results before they get added to the list. 😄